deno/rest/v10/interactions.ts

```diff
@@ -39,30 +40,27 @@ export type RESTGetAPIApplicationCommandsResult = APIApplicationCommand[];
  */
 export type RESTGetAPIApplicationCommandResult = APIApplicationCommand;
 
-type RESTPostAPIBaseApplicationCommandsJSONBody = AddUndefinedToPossiblyUndefinedPropertiesOfInterface<
-	Omit<
-		APIApplicationCommand,
-		| 'id'
-		| 'application_id'
-		| 'description'
-		| 'type'
-		| 'version'
-		| 'guild_id'
-		| 'name_localized'
-		| 'description_localized'
-		| 'default_member_permissions'
-	> &
-		Partial<Pick<APIApplicationCommand, 'default_member_permissions'>>
->;
+type RESTPostAPIBaseApplicationCommandsJSONBody = Omit<
```

Comment:
Oh crap, this is not gonna work the same way because it will not add `| undefined` to these

Reply:
can just use DeepPartial here

Reply:
nevermind, I see what you mean. guess we need to re-introduce the type